python/CMakeLists.txt

```diff
@@ -21,7 +21,17 @@ declare_mlir_python_sources(MPACTPythonSources.PublicAPI
   ADD_TO_PARENT MPACTPythonSources
   SOURCES
     mpactbackend.py
-  )
+)
+
+declare_mlir_python_sources(MPACTPythonSources.SampleModels
+  ROOT_DIR "${MPACT_PYTHON_ROOT_DIR}"
+  ADD_TO_PARENT MPACTPythonSources
+  SOURCES_GLOB
+    models/gcn.py
```

Comment:
Will models/*.py cause any issue?

Reply:
most likely not, but I don't want to do another clean rebuild today ;-(

Reply:
NVM, updated